### ISSUE TITLE
intervals?late_age=0&early_age=130 returns all data

### ISSUE THREAD

**Issue by cambro** [UW-Macrostrat/macrostrat-api#216]
late_age and/or early_age parameters not honored and all data returned with: 
https://macrostrat.org/api/v2/defs/intervals?late_age=0&early_age=130

**Comment by davenquinn** [comment]
This should be added as a test once #232 is finished.

**Comment by davenquinn** [comment]
Low priority perhaps, but this is still a bug.
